lib/Doctrine/ORM/OptimisticLockException.php

```diff
@@ -17,7 +17,7 @@ class OptimisticLockException extends ORMException
 
     /**
      * @param string $msg
-     * @param object $entity
+     * @param ?object $entity
```

Comment:
Please use `object|null` here, `?` is invalid in phpDoc. (It will become a native type later.)

Reply:
Absolutely, good to know